hooks/pre/10-reset_data.rb

```diff
@@ -5,7 +5,7 @@ def reset
   stop_services
   start_services(['postgresql']) if local_postgresql?
   empty_db_in_postgresql('foreman') if foreman_server?
-  reset_candlepin if module_enabled?('katello')
+  reset_candlepin if katello_enabled?
```

Comment:
Just thinking out loud: do we also want to start to prepare for an optional candlepin and introduce `candlepin_enabled?`?

Reply:
I thought about that with my prior PR but we do not have a clean way to detect that I could think of. So I kept it to a clearer representation of how things are.

Reply:
Well, for now we just have it equal to `katello_enabled?` but it prepares the rest of the codebase. Once we circle back to https://github.com/theforeman/foreman-installer/pull/421 it will be good.